lib/davinci_dtr_test_kit/payer_server_groups/adaptive_form_questionnaire_expressions_test.rb

```diff
@@ -12,9 +12,10 @@ class PayerAdaptiveFormExpressionsTest < Inferno::Test
 
     run do
       skip_if retrieval_method == 'Static', 'Performing only static flow tests - only one flow is required.'
-      skip_if scratch[:adaptive_responses].nil?, 'No questionnaire bundle returned.'
-      questionnaire_items_test(scratch[:adaptive_responses], final_cql_test: false)
-      scratch[:adaptive_responses] = nil
+      skip_if scratch[:adaptive_questionnaire_bundles].nil?, 'No questionnaire bundle returned.'
+      questionnaires = extract_questionnaires_from_bundles(scratch[:adaptive_questionnaire_bundles])
+      verify_questionnaire_items(questionnaires)
+      scratch[:adaptive_questionnaire_bundles] = nil
```

Comment:
I agree, but this is unchanged from before (the line is changed just because I renamed the scratch object). I believe this is necessary for the tests to flow correctly. I may be missing something, but I think ultimately we could and should be using outputs instead of scratch for this. Potentially something to look into in the future

Reply:
sounds good - we can look at in the future as time allows